src/main/java/frc/robot/systems/AutoHandlerSystem.java

```diff
@@ -0,0 +1,116 @@
+package frc.robot.systems;
+
+public class AutoHandlerSystem {
+	/* ======================== Constants ======================== */
+	// Auto FSM state definitions
+	public enum AutoFSMState {
+		STATE1,
+		STATE2,
+		STATE3
+	}
+	public enum AutoPathIndex {
```

Comment:
Maybe just `AutoPath`? Index is a bit unclear since we also have numeric indices in `currentStateIndex`.